R/labelled.R

```diff
@@ -96,8 +97,10 @@ make_labelled <- function(supertbl, format_labels = NULL) {
 
   event_labels <- c(
     redcap_event = "Event Name",
+    event_name = "Event Description",
     redcap_arm = "Arm Name",
-    arm_name = "Arm Description"
+    arm_name = "Arm Description",
+    repeat_type = "Repetition Type"
```

Comment:
Have we ever used the phrase "Repetition Type" before? I might just opt for leaving it as "Repeat Type"

Reply:
Changed in the last commit!